### PR TITLE
Deep Learning Assignment 1 & Lab 1 & 2

### PR DESCRIPTION
**Pull request by c5ho**
## What changes are you trying to make? (e.g. Adding or removing code, refactoring existing code, adding reports)
Complete Deep Learning Assignment 1
## What did you learn from the changes you have made?
I learned how to perform hyperparameter tuning, add Convolutional Neural Networks to a model, and how to run experiments.
## Was there another approach you were thinking about making? If so, what approach(es) were you thinking of?
With more time, I would run additional experiments with additional hyperparameter tuning and other regularization techniques.
## Were there any challenges? If so, what issue(s) did you face? How did you overcome it?
I had challenges figuring out with activation functions worked with which loss functions.
## How were these changes tested?
The changes were tested in the notebook through running experiments.
## Checklist
- [x] I can confirm that my changes are working as intended

### CODE REVIEW

**Review by tianyi21 — APPROVED** [review]
Hi Calvin,

Thank you for submitting your DL week 1 assessments. Your A1, Lab1, and Lab2 are now complete. 

Please kindly note that your PR was created after the A1 deadline. While there will be no marks deducted for this assessment, please try to submit the Week 2 assessment on time.

Thanks,
Tianyi [LS]
